apps/veil/src/pages/tournament/ui/landing-card/voting-footer.tsx

```diff
@@ -72,8 +83,11 @@ export const VotingFooter = observer(({ isBanned }: { isBanned: boolean }) => {
           </Text>
         </div>
         <div className='flex gap-2'>
-          <Button actionType='accent' icon={WalletMinimal}>
-            Connect Prax Wallet
+          <Button actionType='accent'>
+            <div className='flex items-center gap-2 whitespace-nowrap'>
```

Comment:
I feel like `whitespace-nowrap` should be applied to all buttons, what do you think? (also cc @VanishMax )

Reply:
Agree that it should be there by default, even no prop needed to disable

Reply:
well, i'd be hesitant to add `truncate` by default, but as a prop it might work well